server/memo_comment.go

```diff
@@ -0,0 +1,191 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	"github.com/usememos/memos/api"
+	apiv1 "github.com/usememos/memos/api/v1"
+	"github.com/usememos/memos/common"
+	"github.com/usememos/memos/store"
+)
+
+func (s *Server) registerMemoCommentRoutes(g *echo.Group) {
+	g.GET("/memo/comment/:memoId", func(c echo.Context) error {
```

Comment:
Maybe `/memo/:memoId/comment` will be better?

Reply:
ok,finished